src/Engine/RadiumEngine.hpp

```diff
@@ -198,6 +275,16 @@ class RA_ENGINE_API RadiumEngine
 
     /// For internal resources management in a filesystem
     std::string m_resourcesRootDir;
+
+    Scalar m_dt;                ///< The time delta between 2 consecutive frames.
+    Scalar m_startTime{0_ra};   ///< The `start` time for the time window.
+    Scalar m_endTime;           ///< The `end` time for the time window.
+    Scalar m_time{0_ra};        ///< The current time.
```

Comment:
It could be more easy to maintain if strongly related members are packed into a local struct. So the above time related members might be packed together ?